components/chassis.py

```diff
@@ -114,10 +114,8 @@ def __init__(
         )
 
         # configuration for motor pid and feedforward
-        self.drive_pid = (
-            Slot0Configs().with_k_p(0.026450530596285438).with_k_i(0).with_k_d(0)
-        )
-        self.drive_ff = SimpleMotorFeedforwardMeters(kS=0.18877, kV=2.7713, kA=0.18824)
+        self.drive_pid = Slot0Configs().with_k_p(0.046062).with_k_i(0).with_k_d(0)
```

Comment:
I presume these gains assume the units are m/s, so we'll need to triple-check our units are correct. Would be worth checking the logs to see if the distance travelled seems reasonable.

Reply:
Yep, I would say this has #65 as a prerequisite